src/app/campaign/inventory/inventory.component.ts

```diff
@@ -10,7 +10,7 @@ import { Flight, InventoryRollup } from '../store/models';
       <p>Please select Start and End Dates, Series, and Zones to view inventory.</p>
     </div>
     <ng-template #inventory>
-      <grove-goal-form [flight]="flight" (goalChange)="goalChange.emit($event)"></grove-goal-form>
+      <grove-goal-form></grove-goal-form>
```

Comment:
Well that's cleaner!

Reply:
Yes, it's just kind of a way to organize the child templates/form controls at this point. Much better. We might want to use ControlValueAccessors with the more complex fields to keep that code together. But for simple form elements, these are just basically child templates now and just need to handle their own display.